File: agora-easycall-ios-oc/AgoraSample/Login/AGSLaunchViewController.m
//
//  AGSLaunchViewController.m
//  AgoraSample
//
//  Created by Shilong on 9/24/15.
//  Copyright (c) 2015 Agora. All rights reserved.
//

#import "AGSLaunchViewController.h"
#import "FOWebViewController.h"
@interface AGSLaunchViewController ()
@property (weak, nonatomic) IBOutlet UIButton *useKeyButton;
@property (weak, nonatomic) IBOutlet UIButton *noKeyButton;

@end

@implementation AGSLaunchViewController
- (IBAction)didClickNoKeyButton:(id)sender
{
    FOWebViewController *web = [[FOWebViewController alloc] initWithURL:@"http://www.agora.io" title:@"Agora"];
    [self.navigationController pushViewController:web animated:YES];
}

- (void)viewWillAppear:(BOOL)animated
{
    [super viewWillAppear:animated];
    self.navigationController.navigationBarHidden = YES;
    
    [self.useKeyButton setTitle:NSLocalizedString(@"Login with Key", @"") forState:UIControlStateNormal];
    [self.noKeyButton setTitle:NSLocalizedString(@"Need a key? Click Here", @"") forState:UIControlStateNormal];
}
@end

File: agora-easycall-ios-swift/SampleSwift/AGSLaunchViewController.swift
//
//  AGSLaunchViewController.swift
//  SampleSwift
//
//  Created by let on 10/15/15.
//  Copyright © 2015 Agora. All rights reserved.
//

import UIKit

class AGSLaunchViewController: UIViewController {
    
    @IBOutlet var useKeyButton: UIButton!
    @IBOutlet var noKeyButton: UIButton!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        // Do any additional setup after loading the view.
    }

    override func viewWillAppear(animated: Bool) {
        super.viewWillAppear(animated)
        navigationController?.navigationBarHidden = true
        
        useKeyButton.setTitle(NSLocalizedString("Login with Key", comment:""), forState:UIControlState.Normal)
        noKeyButton.setTitle(NSLocalizedString("Need a key? Click Here", comment:""), forState:UIControlState.Normal)
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    @IBAction func didClickNoKeyButton(sender: AnyObject) {
        let controller = FOWebViewController(URL: "http://www.agora.io", title: "Agora");
        navigationController?.pushViewController(controller, animated: true)
    }
}
